### ISSUE TITLE
Gateway checker blocked on ipns.co domains

### ISSUE THREAD

**Issue by BigLep** [ipfs/public-gateway-checker#433]
Both fail for me currently:

https://ipfs-public-gateway-checker.on.fleek.co/
https://ipfs.github.io/public-gateway-checker/

> Attackers on bafybeibwzifw52ttrkqlikfzext5akxu7lz4xiwjgwzmqcpdzmp3n5vnbe.ipfs.ipns.co may trick you into doing something dangerous like installing software or revealing your personal information (for example, passwords, phone numbers, or credit cards).

<img width="784" alt="image" src="https://github.com/ipfs/public-gateway-checker/assets/85411/408a83ec-9e70-4bb0-813d-5f1629652169">

I filed a separate issue given there isn't automation in place: https://github.com/ipfs/public-gateway-checker/issues/395

My browser info:

Brave | 1.52.129 Chromium: 114.0.5735.198 (Official Build) (arm64)
-- | --
Revision | c3029382d11c5f499e4fc317353a43d411a5ce1c-refs/branch-heads/5735@{#1394}
OS | macOS Version 13.4.1 (Build 22F82)
JavaScript | V8 11.4.183.25
User Agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36
